functions/_fzf_wrapper.fish

```diff
@@ -4,9 +4,9 @@ function _fzf_wrapper --description "Prepares some environment variables before
     # Use --function so that it doesn't clobber SHELL outside this function.
     set -f --export SHELL (command --search fish)
 
-    # If FZF_DEFAULT_OPTS is not set, then set some sane defaults.
+    # If FZF_DEFAULT_OPTS or FZF_DEFAULT_OPTS_FILE is not set, then set some sane defaults.
     # See https://github.com/junegunn/fzf#environment-variables
-    if not set --query FZF_DEFAULT_OPTS
+    if not set --query FZF_DEFAULT_OPTS || not set --query FZF_DEFAULT_OPTS_FILE
```

Comment:
Oh I think this is wrong. We will enter this branch if either FZF_DEFAULT_OPTS or FZF_DEFAULT_OPTS_FILE are not set, even if the other one IS set. I'll fix it. This is probably why the Linux tests are failing.